frontend/public/index.html

```diff
@@ -26,6 +26,9 @@
     <link rel="preconnect" href="https://maps.googleapis.com" />
     <link rel="preconnect" href="https://www.google-analytics.com" />
 
+    <!-- Preconnect to required origins to speed up load times. -->
```

Comment:
Hmm, this isn't for preconnecting, right?

Reply:
oops I think I didn't save, thanks!